src/main/java/com/redhat/quarkus/mandrel/collector/report/model/RunnerInfo.java

```diff
@@ -29,25 +30,21 @@
 import jakarta.persistence.InheritanceType;
 import jakarta.persistence.Table;
 
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Comment:
Oh I didn't know about this. Cool!

We should probably follow the same approach in the rest of the models and avoid [using `@JsonProperty` to map camelCase java names to snake_case json fields](https://github.com/Karm/collector/blob/main/src/main/java/com/redhat/quarkus/mandrel/collector/report/model/ClassStats.java#L44-L49)

Reply:
I'd swear it was possible to set it globally at some point. I'll try to look it up. 
Feels like I spend too much time wrangling Jenkins and getting rusty on good old Java REST CRUD :)